site/docs/components/navigation-item/usage.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### How to use
+
+#### When to use Navigation Item
+
+- Use Navigation Item when you have multiple levels of navigation in your application or website.
+
+- We suggest first considering a horizontal group of Navigation Items for distinct webpages or applications. If you have multiple (more than eight) services to navigate to at a platform level, or have navigation across distinct apps, consider using a vertical group instead.
+
+- If you need your items to remain visible on scroll, we recommend displaying Navigation Items in a fixed App Header (or header area)
+
+#### When not to use Navigation Item
+
+- When you need to show different views of content within the same page, consider using [Tabs](?path=/docs/documentation-lab-tabs--page) instead.
+
+- When your content is very specific. In this case, a minimalistic or single-page design may be more appropriate than a complex navigation structure. Consider using the [Interactable Card](?path=/docs/documentation-core-card--page), or a [Link](?path=/docs/documentation-core-link--page) component within the body of the page.
+
+### Import
+
+To import [Badge](?path=/docs/documentation-lab-badge--page) from the lab Salt package, use:
+
+```
+import { NavigationItem  } from "@salt-ds/lab";
+```
+
+### Content
+
+- To make the most of screen real estate, our recommendation is that all description text should be clear and concise.
+
+- Always write descriptions in sentence case in a Navigation Item.
+
+- When using a [Badge](?path=/docs/documentation-lab-badge--page) component within a Navigation Item, we recommend using sentence case.
```

Comment:
these component links need updating

Reply:
is it to the examples pages now? (components/badge/examples)